src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskTypesService.java

```diff
@@ -51,7 +51,7 @@ public GetTaskTypesResponse getTaskTypes(AccessControlResponse accessControlResp
 
         //get task-type-dmn(s) for jurisdiction
         Set<TaskTypesDmnResponse> taskTypesDmnResponse =
-            dmnEvaluationService.retrieveTaskTypesDmn(jurisdiction, DMN_NAME);
+            dmnEvaluationService.retrieveTaskTypesDmn(jurisdiction, DECISION_KEY_PREFIX);
```

Comment:
Do we do any duplicate checking of the files we get back?  E.g. if 2 get 2 DMNs back and there is the same id in both and say they have different values?

Reply:
no we don't do duplicate checking. unless the service have 2 files with the same id, the latestVersion parameter will filter out the old one.

Reply:
Replaced the list by a set to avoid duplicate files